server/athenian/api/openapi/openapi.yaml

```diff
@@ -1083,6 +1260,91 @@ components:
       items:
         $ref: '#/components/schemas/DeveloperID'
       type: array
+    TeamMember:
+      description: A team member.
+      example:
+        login: github.com/gkwillie
+        id: github|60340680
+        native_id: 60340680
```

Comment:
Let's not add `native_id` to the response, I cannot imagine why it would needed, and you'd better not know [how we get it](https://github.com/athenianco/athenian-api/blob/master/server/athenian/api/auth.py#L350) :smile:

Reply:
hahaha 😂 

Anyway, the only reason I put is to reuse `User` and it's a required property in `User`,

Reply:
ok, then let's not touch it.